HTTPServerProject.Tests/HTTPServerProxyTests/ResponseTests.cs

```diff
@@ -0,0 +1,224 @@
+using HTTPServerReadTests.Streams;
+using HTTPServerProxy.Response;
+using HTTPServerWriteTests.Streams;
+
+namespace HTTPServerProxyTests.Response;
+
+public class ProxyResponseTests
+{
+    [Fact]
+    public void GetTodoPOSTValidResponseTest()
```

Comment:
Why do all these start with `GetTodo`? Is this a name that hung around from previous versions?

Reply:
No, it's just a silly name. It probably would be better to swap out `GetTodo` for `Proxy`